server/events/webhooks/webhooks.go

```diff
@@ -52,14 +52,19 @@ type MultiWebhookSender struct {
 type Config struct {
 	Event          string
 	WorkspaceRegex string
+	BranchRegex    string
 	Kind           string
 	Channel        string
 }
 
 func NewMultiWebhookSender(configs []Config, client SlackClient) (*MultiWebhookSender, error) {
 	var webhooks []Sender
 	for _, c := range configs {
-		r, err := regexp.Compile(c.WorkspaceRegex)
+		wr, err := regexp.Compile(c.WorkspaceRegex)
+		if err != nil {
+			return nil, err
+		}
+		br, err := regexp.Compile(c.BranchRegex)
```

Comment:
Does this become a breaking change? It might be better to use `.*` when passing empty string.

Reply:
No, I tested a `webhooks` config without defining either `workspace-regex` or `branch-regex` and the Slack notification still went through. For example, `regexp.MatchString("", "main")` evaluates to true, so looks like this will still function fine if the user does not set the `branch-regex` in the config.

Reply:
I see, thanks for your confirmation 👍